notebooks/visualize/visualize.ipynb

```diff
@@ -0,0 +1,1034 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    nRows = 10000
</pre><p>You might want to make this a configurable (user entry widget) and maybe default to 1000 to make it a bit quicker (if less accurate)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/mstic-dataexpl/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>